config/rhoai/ocp-4.17-addons/validating_admission_policy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: kueue-validating-admission-policy
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["ray.io"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["rayclusters"]
```

Comment:
Should this be extended to the other resources that are supported, e.g. PyTorchJobs?

Reply:
Initially, this enforcement will be onto RayClusters only, and is targetting 2.17: https://issues.redhat.com/browse/RHOAIENG-15928 

However, the plan afterwards is to include this enforcement onto all other supported workloads: https://issues.redhat.com/browse/RHOAISTRAT-423 

Enforcing on all supported workloads shouldn't take too much time, I'll push towards adding all other supported workloads for 2.18, unless we require it for this upcoming release.

Reply:
Sounds good, thanks for the clarification.

Reply:
Nice!